R/dependencies.R

```diff
@@ -513,37 +513,48 @@ renv_dependencies_discover_description <- function(path,
   if (inherits(dcf, "error"))
     return(renv_dependencies_error(path, error = dcf))
 
-  if (is.null(fields)) {
-    # Most callers don't pass in project so we need to get it from global state
+  fields <- fields %||% {
+
+    # most callers don't pass in project so we need to get it from global state
     project <- project %||%
       renv_dependencies_state(key = "root") %||%
       renv_restore_state(key = "root") %||%
       renv_project_resolve()
 
+    # get fields from settings
     fields <- settings$package.dependency.fields(project = project)
 
     # if this is the DESCRIPTION file for the active project, include
     # the dependencies for the active profile (if any) and Suggested fields.
+    # collect profile-specific dependencies as well
```

Comment:
IMO this lines duplicates the info in the previous line

Reply:
You're right, I removed it.